modules/vba_documents/app/serializers/vba_documents/upload_serializer.rb

```diff
@@ -11,27 +11,25 @@ class UploadSerializer
     set_type :document_upload
     set_id :guid
 
-    attributes :guid, :code, :updated_at
+    attributes :guid, :status, :code
```

Comment:
Fixing the order of fields here, and bringing `status` back into the shared `UploadSerializer`, since there are no changes between V1 and V2. (V2 should be removed at some point, but that task seemed too large for this PR.)